style.css

```diff
@@ -0,0 +1,66 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+body {
+  background-color: lightgrey;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+.calculator {
+  margin-top: 30px;

```

Comment:
You should also try using relative measurements like `em` or `rem` instead of absolute.